capitolweb/workers/crec_parser.py

```diff
@@ -31,6 +34,12 @@
 CREC_PREFIX_TEMPLATE = 'crec/%Y/%m/%d/crec'
 CREC_KEY_TEMPLATE = '{prefix}/{id}.htm'
 
+APPROX_MATCH_THRESHOLD = 90
+GENERIC_SPEAKERS = [
+    'The CLERK', 'The SPEAKER pro tempore', 'The SPEAKER',
+    'The PRESIDING OFFICER', 'The ACTING PRESIDENT pro tempore'
+]
```

Comment:
we may want to update this list with some more from https://github.com/propublica/Capitol-Words/blob/master/parser/parser.py#L169 and https://github.com/propublica/Capitol-Words/blob/master/parser/parser.py#L175

Reply:
That's awesome, thank you! Will add the stuff to the list.